.github/pull_request_template.md

```diff
@@ -1,6 +1,10 @@
+## Problem
+
+*Please describe or link to an issue about the problem you're solving*
```

Comment:
I think we could pull in a light checklist here to help everyone know how to describe a problem. We could even pull in the statement from the RFC that's in review.

> Problem statement: Who are we building for, what are their needs, why is this important?

I don't want to ask for too much info up front, but it could also be good to communicate any assumptions, validation, technical or business constraints, or other discovery work that informs the problem. We don't need all of it, but if it's available, including it here would be great.

Some **examples** of discovery details that clarify a problem:
- The problem only affects specific roles
- The problem only affects specific plans or hosting types
- There's a upcoming social or business obligation that requires we launch by a deadline
- There's an underlying technical constraint that prevents us from achieving our ideal solution in the allocated sprint time
- Example data sets that quantify the problem in terms of real product constructs (we're great at this already, but I want us to keep doing it)

Reply:
We had a discussion about checklists at the offsite. The general consensus was the checklists initially get used then ultimately got ignored so don't end up adding value in the medium / long term.

We aligned on adding a link to coding conventions for people who want a smoother review. It could be valuable to add a link to context around describing a problem. But since most problems are initially filed as issues (i.e. bugs or feature requests) - it might be worth putting our energy into that template, and using this to reference to that issue, what do you think?

Reply:
A one-sentence prompt like "Who are we building for, what are their needs, why is this important?" is good since it makes you think "one level deeper" more, but I agree that PR checklists aren't very effective.

Reply:
```suggestion
*Who are we building for, what are their needs, why is this important?*
```

Reply:
Sounds good. My goal is to prompt us to think one level deeper, so I'm happy as long as we solve for that. 👍